monai/bundle/workflows.py

```diff
@@ -166,18 +168,18 @@ class ConfigWorkflow(BundleWorkflow):
     For more information: https://docs.monai.io/en/latest/mb_specification.html.
 
     Args:
-        run_id: ID name of the expected config expression to run, default to "run".
-            to run the config, the target config must contain this ID.
+        config_file: filepath of the config file, if it is a list of file paths, the content of them will be merged.
+        meta_file: filepath of the metadata file, if it is a list of file paths, the content of them will be merged.
+            If None, default to "configs/metadata.json", which is commonly used for bundles in MONAI model zoo.
```

Comment:
```suggestion
        config_file: filepath of the config file, if this is a list of file paths, their contents will be merged in order.
        meta_file: filepath of the metadata file, if this is a list of file paths, their contents will be merged in order.
            If None, default to "configs/metadata.json", which is commonly used for bundles in MONAI model zoo.
```

I think the ordering is important to highlight here.